### PR TITLE
Add classname to browser title

### PR DESCRIPTION
**Pull request by paulovitin**
Title of browser with class name currently opened #868 

screenshot 2018-10-13 01 04 29

### CODE REVIEW

File: src/dashboard/Data/Browser/Browser.react.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ import styles                             from 'dashboard/Data/Browser/Browser.s
 import subscribeTo                        from 'lib/subscribeTo';
 import * as ColumnPreferences             from 'lib/ColumnPreferences';
 import * as queryString                   from 'query-string';
+import DocumentTitle                      from 'react-document-title';
```
Comment (flovilmart) [inline src/dashboard/Data/Browser/Browser.react.js]: I’m always cautious when adding new dependencies.

Did you consider Helmet? It is well maintained and popular.

**Comment by paulovitin** [comment]
@flovilmart take a look. What you thing about put the sections in the title too? `Jobs - Parse Dashboard`, `Config - Parse-Dashboard`?

**Comment by paulovitin** [comment]
@flovilmart need something more here?

**Comment by flovilmart** [comment]
Thansk!!

**Pull request merged**